commands/raid.js

```diff
@@ -1,6 +1,6 @@
 ﻿module.exports.execute = async (client, message) => {
 	return await message.channel.send(
-		'**RAAAAAAAAAAAAAAAAAAAID⚔** with ' + message.author + '\nhttps://cuckoo.team/koa',
+		'**RAAAAAAAAAAAAAAAAAAAID⚔** with ' + `${message.author}` + '\nhttps://cuckoo.team/koa',
```

Comment:
nitpick: can this be `<@${message.author.id}>` instead to prevent any future changes that may occur?